Dockerfile

```diff
@@ -0,0 +1,45 @@
+# To build :
+# $ docker build -t btcgpu:latest .
+# To run :
+# docker run -v /home/user/btcgold:/home/bitcoingold/.bitcoingold -d btcgpu:latest 
+
+FROM debian:sid as builder
+
+RUN apt-get update && apt-get install -y gnupg && apt-key adv --keyserver keyserver.ubuntu.com --recv-keys D46F45428842CE5E
+RUN echo "deb http://ppa.launchpad.net/bitcoin/bitcoin/ubuntu xenial main" > /etc/apt/sources.list.d/bitcoin-ubuntu-bitcoin-xenial.list
+
+RUN apt-get update && apt-get install -y git build-essential libtool autotools-dev automake pkg-config libssl-dev libevent-dev bsdmainutils
+
+RUN apt-get install -y libdb4.8-dev libdb4.8++-dev
```

Comment:
Not sure if you are separating these on purpose but they should all be `&&`ed from the apt-get update to avoid a split brained apt repo which will lead to confusing broken packages one day down the line.

Reply:
yeah we can make one line :)